com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -581,6 +581,8 @@ public static List<CommandLineOption> getCommandLineOptions() {
 
                 opt("l", "liveupdate", ONE, "Yes if liveupdate content should be published", true),
 
+                opt(null, "with-sha1", ZERO, "Generate (and verify) sha1 signatures from build artifacts", false),
```

Comment:
How will a user know this only applies to html5 builds?

Reply:
Added note to description